TopSection Update

rayhm6419 opened the pull request:
1. This version would change the font suddenly change from 1339px to 1440p, my idea is to try to make it consistent compared to the old version when the screen size gets smaller

- use px instead of vw for font size

For example when the screen size at 800px
**updated**
![image](https://github.com/1thing-org/1thing.org/assets/105839955/2cf19597-df0c-4c5c-93ef-14aad98727a4)

**old**
![image](https://github.com/1thing-org/1thing.org/assets/105839955/d603ae80-8848-40a7-b910-b39530f048fc)

2. The Take Action button linked to Take Action page
3. Mached all the font to san-serif
4. Fix Indentation 
5. Remove this section from Testimonial page
![image](https://github.com/1thing-org/1thing.org/assets/105839955/ba0def60-5b25-4d8f-ab34-a4f34541bf12)

lima01 commented on src/components/hero/HeroSection.tsx:
```diff
@@ -16,60 +16,81 @@ function HeroSection(props: Props) {
+                  textTransform: "uppercase",
+                }}
+                onClick={() => {
+                  console.log("Button clicked!");
```
no need to log anything here.

lima01 reviewed (commented):
I found there are a lot more changes in the PR that are not covered by the description. Shall we split them into different PRs?

rayhm6419 commented:
> I found there are a lot more changes in the PR that are not covered by the description. Shall we split them into different PRs?

Mainly, the changes are in the section's font, position, and code indentation. Also, I switched san-serif to the default font. I think this pull request is ready for merging.
